### DISCUSSION TITLE
How do I bulk change from Junicode Two Beta to Junicode 2 Final in Word documents?

### DISCUSSION THREAD

**Discussion by Floschi12345** [psb1558/Junicode-font#264] [General]
As stated in the title: How do I bulk change from Junicode Two Beta to Junicode 2 Final in Word documents? Our research project started using Unicode Two Beta last year using it for translations and general text production. Now that the final version of Junicode Two is released we would like to change this final version. The problem is: We have over a hundred .docx documents of varying lengths using several font weights of Junicode Two Beta. Is there any easy, bulk way to change all these documents to Junicode Two Final? Any help would be greatly appreciated! Thanks and Cheers and a Happy New Year!

**Comment by psb1558** [comment]
I *think* that the character set and OpenType features have been pretty stable since I renamed the font "Junicode Two Beta," so it should be a search-and-replace of font names only. I am very far from an expert on MS Word (I don't even have easy access to a copy), but I'm pretty sure that the "Find and Replace" dialog lets you do exactly that. If you can do it from a dialog box, you can do it with a VBA script, and if you can do it with a script, it ought to be possible to do it in bulk.

[Here](https://answers.microsoft.com/en-us/msoffice/forum/all/batch-editing-ms-word-documents-find-and-replace/68cce53b-3771-425a-9f77-bfed78087478) is an article from MS support on "Batch Editing." The question also came up on [StackExchange](https://superuser.com/questions/1010991/how-to-batch-find-n-replacein-doc-files), and on [Quora](https://www.quora.com/Can-you-find-and-replace-across-multiple-Word-documents). If you have a budget, there's at least one utility that claims to be able to do it (but I can't find the link just now).
